Formula/dust.rb

```diff
@@ -4,6 +4,7 @@ class Dust < Formula
   url "https://github.com/bootandy/dust/archive/v0.8.2.tar.gz"
   sha256 "890972fbf1a7f0a336c0f20e1e9ecc756c62d3debd75d22b596af993a3d8af01"
   license "Apache-2.0"
+  revision 1
```

Comment:
`revision` bump is a bit much for installing completions.

Reply:
I thought it's fine as I did this in past PRs as well. Not bumping revisions means users will have to wait for next version, which can be months later or who knows when. If it's a regularly updated or there's pending version bump I agree it's not necessary.

Reply:
`revision` bumps lead to mandatory reinstalls with `brew upgrade`, and that can turn out to be a pretty expensive operation. Minor formula improvements are, to me, not important enough to incur that cost, since users who are interested in the minor improvement can do `brew reinstall`.

Typically I'd err on restricting `revision` bumps to cases only where pre-existing installations would be broken without it.